.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,14 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Helper-Scripts Community Support
-    url: https://github.com/community-scripts/ProxmoxVE/discussions
-    about: Please ask and answer questions here.
+    - name: 📃 Documentation Issue
+      url: https://github.com/community-scripts/ProxmoxVE/issues/new?assignees=&labels=Status%3A+Unconfirmed&projects=&template=frontend_report.yml&title=%5BFrontend%5D%3A+
+      about: This issue tracker is not for documentation issues. Please file documentation issues here.
+    - name: 🤔 Questions and Help
+      url: https://github.com/community-scripts/ProxmoxVE/discussions
+      about: This issue tracker is not for support questions. Please refer to the community's help.
```

```suggestion
      about: For suggestions or questions, please use the Discussions section.
```
consistent with descriptions in template